docs/index.md

```diff
@@ -190,3 +190,246 @@ The last step is to calculate all the specified corrections for the main fake fa
 ## Hints
 * check out `configs/general_definitions.py`, this file has many relevant definition for plotting (dictionaries for names) and correctionlib output information
 * check `ntuple_path` and `output_path` (preselection) and `file_path` and `workdir_name` (fake factors, corrections) in the used config files to avoid wrong inputs or outputs
+
+## Automated Equipopulated Binning
+
+The framework includes a utility script, `adjust_binning.py`, for an automatic calculation and update binning definitions within the configuration in case of equipopulated binning approach of specified variable (`var_dependence`) across various categories for a more robust statistical treatment in each analysis region. The general intention is to have an equipopulated data distribution in the SRlike region given possible additional splittings (i.e. in `njets`).
+
+The script is intended to be run berofe the fake factor and correction calculation by using
+
+```bash
+python adjust_binning.py --config config.yaml --cut-config config_with_cuts.yaml --processes QCD Wjets --cut-region ARlike
```

Comment:
`config_with_cuts.yaml` -> `config_with_region_cuts.yaml` would be a bit more descriptive